plugins/woocommerce/includes/class-woocommerce.php

```diff
@@ -244,7 +244,7 @@ private function init_hooks() {
 		register_shutdown_function( array( $this, 'log_errors' ) );
 
 		add_action( 'plugins_loaded', array( $this, 'on_plugins_loaded' ), -1 );
-		add_action( 'plugins_loaded', array( $this, 'init_jetpack_connection_config' ), 1 );
+		add_action( 'init', array( $this, 'init_jetpack_connection_config' ), 1 );
```

Comment:
From [the docs](https://github.com/Automattic/jetpack/tree/391f1f22abbc8ba9c34344d14e659ecaf92df936/projects/packages/config#usage), it looks like this one needs to be called "at or before plugins_loaded". So we can't change this to `init`. Does this mean the "WooCommerce" string in the linked callback shouldn't be translated?

Reply:
Either that, or that config package is updated to work on `init` instead or receive a callback or something (so that the translation function isn't immediately invoked)